tests.py

```diff
@@ -128,6 +155,46 @@ def test_status(self):
 
         # Make sure mock_send_receive_set_timer() has been called twice with correct parameters
         self.assertEqual(result['test_result'], "SUCCESS")
+        
+    def test_set_colour(self):
+        d = pytuya.BulbDevice('DEVICE_ID_HERE', 'IP_ADDRESS_HERE', LOCAL_KEY)
+        d._send_receive = MagicMock(side_effect=mock_send_receive_set_colour)
+        
+        result = d.set_colour("ffffff")
```

Comment:
not sure this works with the API change (sorry, I've not had chance to pull and test)?

Reply:
For some reason I accidentally kept the test for the old API. Will remove this one and keep the new one.